sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -300,21 +300,30 @@ cc_test(
 # MASK ROM E2E TESTS
 
 # Bootup tests
-BOOT_FAILURE_MSG = "BFV:[0-9a-z]{{8}}\r\nLCV:[0-9a-z]{{8}}\r\n"
+BOOT_FAILURE_MSG = "BFV:[0-9a-f]{8}"
+
+BOOT_SUCCESS_MSG = "PASS!"
 
 opentitan_functest(
     name = "e2e_bootup_success",
     srcs = ["mask_rom_test.c"],
     cw310 = cw310_params(
         bitstream = "//hw/bitstream:mask_rom",
-        tags = ["broken"],
+        exit_failure = BOOT_FAILURE_MSG,
+        exit_success = BOOT_SUCCESS_MSG,
     ),
+    manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
     signed = True,
     verilator = verilator_params(
+        exit_failure = BOOT_FAILURE_MSG,
+        exit_success = BOOT_SUCCESS_MSG,
         rom = ":mask_rom_sim_verilator_scr_vmem",
+        # Note: Leaving as broken since this test takes a long time and
+        # we don't really need it as long as the CI runs the FPGA version.
         tags = [
             "broken",
         ],
+        timeout = "long",
     ),
     deps = [
         "//sw/device/lib/testing/test_framework:ottf_main",
```

Comment:
I cannot take credit for this (since @milesdai added this test) but I like how it doesn't depend on rom_ext and we can test mask_rom independent of it. I think we will have quite a few e2e tests with rom_ext dependencies, so I'd like to keep this as is if you don't mind.

Reply:
I do worry about implicit dependencies because downstream consumers will probably have a hard time understanding how things are integrated, but an alternative option is to make sure we document the flow. Integrators will be looking at how to plugin OpenTitan with their existing infrastructure to run things as system verification with code signing etc.